analysis_tasks/get_video_watching_durations.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+import json
+import psycopg2
+
+
+# Gets datetime object from log 'time' string
+def parse_datetime(log_timestamp):
+    dt = log_timestamp.split("T")
+    date = dt[0].split("-")
+    time = dt[1].split(".")[0].split(":")
+    return datetime.datetime(int(date[0]), int(date[1]), int(date[2]), hour=int(time[0]), minute=int(time[1]), second=int(time[2]))
+
+
+# Prints total time of video-watching for every user (as a table)
+def print_result_table(user_times):
+    print("username | time (sec)")
+    for username in user_times:
+        print(username, "|", user_times[username].total_seconds())
```

Comment:
May be we should put results into .csv file? We've got special directory for that :-) ```libs/result/```

Reply:
Ok, i haven't noticed that :)
So i'll add this feature